init.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import zipfile
+import io
+import os
+import sys
+
+repos_url = "https://api.github.com/users/"
+repo_zip_url = "https://github.com/"
+
+def getRepos(username=None):
+    print ("Getting repository list")
+    try:
+        url = repos_url + username + "/repos"
```

Comment:
Variable name is inaccurate here: "username"
```suggestion
        url = repos_url + organization + "/repos"
```
or something more hard-coded like:

```suggestion
        url = "https://github.com/bitnaughts/repos"
```